File: ployglots/mta-ployglot-pkg-vault/packages/ts/examples/sdk/01-load-from-env.mjs
import { VaultFileSDK } from '@polyglot/vault-file';

const sdk = VaultFileSDK.create().withEnvPath('.env').build();
const result = sdk.loadConfig();
if (!result.success) {
  console.error('load failed:', result.error);
  process.exit(1);
}
console.log(JSON.stringify(result.data));

File: ployglots/mta-ployglot-pkg-vault/packages/py/examples/sdk/01_load_from_env.py
from polyglot_vault_file import VaultFileSDK

sdk = VaultFileSDK.create().with_env_path(".env").build()
result = sdk.load_config()
if not result.success:
    print(f"load failed: {result.error}")
    raise SystemExit(1)
print(result.data.model_dump_json(by_alias=True))
